internal/pkg/skuba/kubeadm/configmap_test.go

```diff
@@ -367,10 +367,7 @@ func TestUpdateClusterConfigurationWithClusterVersion(t *testing.T) {
 				}
 				initCfg.APIServer.ControlPlaneComponent.ExtraArgs["enable-admission-plugins"] = currentAdmissionPlugins
 			}
-			_, err := UpdateClusterConfigurationWithClusterVersion(&initCfg, tt.clusterVersion)
-			if err != nil {
-				t.Errorf("expected no error but an error returned: %v", err)
-			}
```

Comment:
This seems odd to remove.  Is an error expected in the old version of k8s? :)

Reply:
Ok, I see that the function it calls returns differently when the commit is reverted.  I guess this makes sense then.